piker/ui/_exec.py

```diff
@@ -124,6 +181,40 @@ def closeEvent(
         # raising KBI seems to get intercepted by by Qt so just use the system.
         os.kill(os.getpid(), signal.SIGINT)
 
+    @property
+    def status_bar(self) -> QStatusBar:
+
+        # style and cached the status bar on first access
+        if not self._status_bar:
+            # TODO: i guess refactor stuff to avoid having to import here?
+            from ._style import _font_small, hcolor
+            sb = self.statusBar()
+            sb.setStyleSheet((
+                f"color : {hcolor('gunmetal')};"
```

Comment:
btw i **really** like this font color for widget text; nice and muted but still matches.